README.md

```diff
@@ -60,6 +60,11 @@ On a rooted Google OnHub, the command needs to be run with the "chronos" user. I
     Password:
     $ curl -Ls git.io/vddgY | bash
 
+Help
+----
+
+Please check out the [wiki](https://github.com/skycocker/chromebrew/wiki) to find out more information about Chromebrew including helpful tips, resource links and frequently asked questions. Also please check existing [issues](https://github.com/skycocker/chromebrew/issues) before submitting a new [pull request](https://github.com/skycocker/chromebrew/pulls).
```

Comment:
Remove `[pull request](https://github.com/skycocker/chromebrew/pulls)` and replace with `issue`.  Also, I agree with @JL2210 that italics are not necessary here.